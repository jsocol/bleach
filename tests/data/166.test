<div id="97"><!-- IE 5-9 -->
<div id=d><x xmlns="><iframe onload=alert(97)"></div>
<script>d.innerHTML+='';</script>
<!-- IE 10 in IE5-9 Standards mode -->
<div id=d><x xmlns='"><iframe onload=alert(2)//'></div>
<script>d.innerHTML+='';</script>//["'`-->]]>]</div>
