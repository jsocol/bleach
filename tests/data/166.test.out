<div id="97">
<div id="d"></div>


<div id="d"></div>
//["'`--&gt;]]&gt;]</div>
